playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -1123,6 +1123,10 @@ edxapp_npm_bin: "{{ edxapp_npm_dir }}/bin"
 edxapp_settings: '{{ EDXAPP_SETTINGS }}'
 EDXAPP_NODE_VERSION: "20"
 EDXAPP_NPM_VERSION: "10.7.0"
+
+NPM_ALIASES:
+  "@edx/brand": "npm:@edx/brand-edx.org@~2.0.0"
```

Comment:
This package is available as version `2.1.3` on npm, so we should update it here accordingly.  
Additionally, since it's listed with the caret (`^`) operator in `edx-platform`'s `package.json`, we can follow the same convention and use `^2.1.3` here as well.